src/suit/js/SUIT.js

```diff
@@ -78,10 +74,15 @@ if (tapUrl) { // if a url is produced with confidence put it at the top otherwis
 if (!tapUrl || !confident) {
     setTimeout( () =>
             showInfoPopup(
-                `Could not infer the location of the TAP service for this LSP instance from the window URL: ${window.location.href}`, 5000));
+                `Could not infer the location of the TAP service for this RSP instance from the window URL: ${window.location.href}`, 5000));
 }
 
-var options = {
+let options = {
+    multiTableSearchCmdOptions: [
```

Comment:
@loitly I believe it's this set of options that keeps TAP out of `MultiTableSearchCmd`.  Trey added this at my request because in the current Rubin context it doesn't make sense to bury TAP under a generic catalog-search button.